src/ServiceControl.Audit/Infrastructure/LoggingConfigurator.cs

```diff
@@ -23,81 +21,47 @@ public static void ConfigureLogging(LoggingSettings loggingSettings)
                 return;
             }
 
-            var version = FileVersionInfo.GetVersionInfo(typeof(LoggingConfigurator).Assembly.Location).ProductVersion;
             var nlogConfig = new LoggingConfiguration();
             var simpleLayout = new SimpleLayout("${longdate}|${threadid}|${level}|${logger}|${message}${onexception:|${exception:format=tostring}}");
-            var header = $@"-------------------------------------------------------------
-ServiceControl Audit Version:				{version}
--------------------------------------------------------------";
 
             var fileTarget = new FileTarget
             {
+                Name = "file",
                 ArchiveEvery = FileArchivePeriod.Day,
                 FileName = Path.Combine(loggingSettings.LogPath, "logfile.${shortdate}.txt"),
                 ArchiveFileName = Path.Combine(loggingSettings.LogPath, "logfile.{#}.txt"),
                 ArchiveNumbering = ArchiveNumberingMode.DateAndSequence,
                 Layout = simpleLayout,
                 MaxArchiveFiles = 14,
-                ArchiveAboveSize = 30 * MegaByte
-            };
-
-            var ravenFileTarget = new FileTarget
-            {
-                ArchiveEvery = FileArchivePeriod.Day,
-                FileName = Path.Combine(loggingSettings.LogPath, "ravenlog.${shortdate}.txt"),
-                ArchiveFileName = Path.Combine(loggingSettings.LogPath, "ravenlog.{#}.txt"),
-                ArchiveNumbering = ArchiveNumberingMode.DateAndSequence,
-                Layout = simpleLayout,
-                MaxArchiveFiles = 14,
-                ArchiveAboveSize = 30 * MegaByte
+                ArchiveAboveSize = 30 * megaByte
             };
 
             var consoleTarget = new ColoredConsoleTarget
             {
+                Name = "console",
                 Layout = simpleLayout,
+                DetectConsoleAvailable = true,
+                DetectOutputRedirected = true,
                 UseDefaultRowHighlightingRules = true
             };
 
-            var nullTarget = new NullTarget();
-
-            nlogConfig.AddTarget("console", consoleTarget);
-            nlogConfig.AddTarget("debugger", fileTarget);
-            nlogConfig.AddTarget("raven", ravenFileTarget);
-            nlogConfig.AddTarget("null", nullTarget);
-
             // Always want to see license logging regardless of default logging level
             nlogConfig.LoggingRules.Add(new LoggingRule("Particular.ServiceControl.Licensing.*", LogLevel.Info, fileTarget));
-            nlogConfig.LoggingRules.Add(new LoggingRule("Particular.ServiceControl.Licensing.*", LogLevel.Info, consoleTarget)
-            {
-                Final = true
```

Comment:
So final did instruct to quit processing any following rules when this one matches. Was this some sort of optimization?

Reply:
After investigating, I couldn't see any reason for this to be here, and it didn't have any impact that I could observe.